### PR TITLE
Homework09

### PR DESCRIPTION
**Pull request by Hronologiya**

### CODE REVIEW

File: lesson_09/test_homework_09.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        self.assertEqual(result, expected_result, "The string reversal is incorrect for positive test case.")
+
+    def test_reverse_string_negative(self):
+        input_string = ""  # Порожній рядок
+        expected_result = ""  # Порожній рядок залишається порожнім після реверсу
+        result = reverse_string(input_string)
+        self.assertEqual(result, expected_result, "The string reversal is incorrect for negative test case.")
+
+
+if __name__ == "__main__":
```
Comment (DartBrovsky) [inline lesson_09/test_homework_09.py]: It is no need to create __name__ == __main__ for each class. It is enough to create at least 1 for whole file.

File: lesson_09/test_homework_09.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    unittest.main()
+
+#Задача 2
+
+import unittest
```
Comment (DartBrovsky) [inline lesson_09/test_homework_09.py]: As well it is no need to import each time unittest and other modules for each class separately. It is enough to import all needed dependencies at the start of current file.

**Review by DartBrovsky — APPROVED** [review]
I will approve your homework solution. But leave some comments for your code improvement. Please take a look into them. Thanks.

File: lesson_09/test_homework_09.py (hunk 1 above)
Comment (Hronologiya) [reply lesson_09/test_homework_09.py]: Thank you for your comments; I have fixed everything.

File: lesson_09/test_homework_09.py (hunk 2 above)
Comment (Hronologiya) [reply lesson_09/test_homework_09.py]: Thank you for your comments; I have fixed everything.

**Pull request merged**